docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_winsrv/_index.md

```diff
@@ -84,150 +88,138 @@ By the end of the guide, you will have an Azure VM installed with Windows Server
+- Open Microsoft SQL Server Management Studio (a Windows shortcut will be created for you) and validate the *AdventureWorksLT2019* sample database is deployed.
 
-    ![Screenshot showing SQL Management Studio](./17.jpg)
+    ![Screenshot showing SQL Management Studio](./sql_server_management_login.png)
 
-    ![Screenshot showing SQL Management Studio](./18.jpg)
+    ![Screenshot showing SQL Management Studio](./sql_server_management_data.png)
 
 ## Azure SQL Assessment
 
-Now that you have both the server and SQL projected as Azure Arc resources, the last step is complete the initiation of the SQL Assessment run.
+Now that you have both the server and SQL projected as Azure Arc resources the last step is to complete the initiation of the SQL Assessment run.
 
-* On the SQL Azure Arc resource, click on "Environment Health" followed by clicking the "Download configuration script".
+- On the SQL Azure Arc resource click on "Environment Health", then click "Download configuration script".
 
-    Since the *LogonScript* run in the deployment step took care of deploying and installing the required binaries, you can safely ignore and delete the downloaded *AddSqlAssessment.ps1* file.
+    Since the *installArcAgentSQL* run in the deployment step took care of deploying and installing the required binaries you can safely ignore and delete the downloaded *AddSqlAssessment.ps1* file.
 
     Clicking the "Download configuration script" will simply send a REST API call to the Azure portal which will make "Step3" available and will result with a grayed-out "View SQL Assessment Results" button.
 
-    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./19.jpg)
+    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./arc_sql_resource_portal.png)
 
-    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./20.jpg)
+    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./environment_health_button.png)
 
-    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./21.jpg)
+    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./download_script_button.png)
 
-    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./22.jpg)
+    ![Screenshot showing Environment Health blade of Azure Arc-enabled SQL server](./view_assessment_disabled.png)
 
-    It might take a bit of time, but after ~45-60min you will notice how the "View SQL Assessment Results" button is available for you to click on. At this point, the SQL assessment data and logs are getting injected to Azure Log Analytics.
+    It might take a bit of time, but after approximately 45-60min you will notice how the "View SQL Assessment Results" becomes available for you to click on. At this point the SQL assessment data and logs are getting injected to Azure Log Analytics.
```

Comment:
```suggestion
    It might take a bit of time, but after approximately 45-60 minutes you will notice how the "View SQL Assessment Results" becomes available for you to click on. At this point the SQL assessment data and logs are getting injected to Azure Log Analytics.
```